harness/benchmarks/src/main/kotlin/godot/benchmark/Simple.kt

```diff
@@ -54,20 +54,25 @@ class Simple : Object() {
     }
 
     @RegisterFunction
-    fun benchmarkICall(): Node {
+    fun benchmarkIcall(): Int {
         val node = Node()
         node.getInstanceId()
-        return node
+        node.free()
+        return 0
     }
 
-    fun benchmarkICallWithLoop(): Node {
+    fun benchmarkIcallWithLoop(): Int {
```

Comment:
out of curiosity why the change to returning an int?

Reply:
Can be unit. Fix here is that we free node. Not freeing it was corrupting benchmark as GC was checking a list that was becoming bigger and bigger. That was causing cost augmentation in function of iterations number.